modules/ROOT/pages/event-tracking.adoc

```diff
@@ -186,29 +194,33 @@ services:
 ----
 
 
+=== Complex Tracking Example
+
+This example sets tracking levels and settings at all three levels: global, Mule app, and flow.
 
-=== Setting Complex Tracking Configurations
+Global:
 
-In the example below, tracking levels and settings are overridden at all three levels: global, Mule application, and flow.
+* `globalTrackingLevel` set to `OFF`, which is the default setting if you don't specify `globalTrackingLevel`.
```

Comment:
See Line 26, which states that the default is NONE, not OFF....
This applies to all of the text that follows.

Reply:
Query in to @mcocaro.